WordPress/Classes/Services/SiteSuggestionService.swift

```diff
@@ -15,7 +16,16 @@ class SiteSuggestionService {
     */
     func suggestions(for blog: Blog, completion: @escaping ([SiteSuggestion]?) -> Void) {
 
-        if let suggestions = retrievePersistedSuggestions(for: blog), suggestions.isEmpty == false {
+        let throttleDuration: TimeInterval = 60 // seconds
+        let isBelowThrottleThreshold: Bool
+        if let requestDate = requests[blog] {
+            isBelowThrottleThreshold = Date().timeIntervalSince(requestDate) < throttleDuration
+        } else {
+            isBelowThrottleThreshold = false
+        }
```

Comment:
@jleandroperez, I wish I knew a Swifty way of cleaning this up into a one-liner.

Reply:
I think this is the most readable approach! Anything else would probably involve using `map`, but you'd need to deal with the `nil` scenario as well (which is not cool!!)

```
let isBelowThrottleThreshold = requests[blog].map { date in
      Date().timeIntervalSince(requestDate) < throttleDuration
}  ?? false
```

Reply:
Yeah, using map like this is an alternative. Thanks for the suggestion 👍